geo_json_metrics/vcr_calculator.py

```diff
@@ -0,0 +1,27 @@
+def map_vcr(velocities: list[float]) -> list[float]:
+    """Map the VCR function over each pair from the list, skipping the first element
+    Args:
+        velocities (list[float]): A list of floats corresponding to the velocities/speeds
```

Comment:
Velocity og speed er to forskellige ting, det du mener er speed (fordi enheden er m/s)